chain-extensions/types/xvm/src/lib.rs

```diff
@@ -65,4 +64,6 @@
     pub to: Vec<u8>,
     /// Encoded call params
     pub input: Vec<u8>,
+    /// Value to transfer
+    pub value: Balance,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `value` field has been added to the `XvmCallArgs` struct. LGTM!


<!-- This is an auto-generated comment by OSS CodeRabbit -->